sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 4.1.0-beta.1 (Unreleased)
-
+## 4.1.0 (2020-06-25)
+### New Features
+* Added support for `GROUP BY` query.
+* Increased the default value for RntbdTransportClient.Options.maxChannelsPerEndpoint to 130.
```

Comment:
May be instead of mentioning the `RntbdTransportClient` class here (since it is implementation level detail) - I think we should mention something like this : 

* Increased the default value of `maxConnectionsPerEndpoint` to 130 in `DirectConnectionConfig`
* Increased the default value of `maxRequestsPerConnection` to 30 in `DirectConnectionConfig`

Reply:
updated